relayer/utils/__init__.py

```diff
@@ -0,0 +1,3 @@
+def get_elapsed_time_in_milliseconds(start_time, end_time):
+    elapsed_time = end_time - start_time
+    return elapsed_time.microseconds / 1000.0 + elapsed_time.seconds * 1000
```

Comment:
Read https://docs.python.org/3/library/datetime.html#timedelta-objects

The values are normalised such as:

```
0 <= microseconds < 1000000
0 <= seconds < 3600*24 (the number of seconds in one day)
-999999999 <= days <= 999999999
```

If that isn't clear enough, you can check it on the console:
```>>> from datetime import datetime
>>> t1 = datetime.utcnow()
>>> t2 = datetime.utcnow()
>>> t2 - t1
datetime.timedelta(0, 8, 529430)
>>> td = t2 - t1
>>> td
datetime.timedelta(0, 8, 529430)
>>> td.seconds
8
>>> td.microseconds
529430```

Look on there are `8` seconds on the timedelta but also `529430` microseconds, the microseconds are normalized so you only see them as fractions of a second, if the microseconds accounted for the total of microseconds the number would be something like `8529430` which means, 8 seconds and a half